plugins/view-resources/src/components/EditDoc.svelte

```diff
@@ -45,24 +45,24 @@
   let lastId: Ref<Doc> = _id
   let object: Doc
 
-  const client = getClient()
-  const hierarchy = client.getHierarchy()
+  const pClient = getClient()
```

Comment:
Why was it renamed?

Reply:
since it has client in parameters later in code.